sources/packages/forms/src/form-definitions/studentwelcomepage.json

```diff
@@ -0,0 +1,1813 @@
+                                            "persistent": false,
+                                            "clearOnHide": true,
+                                            "refreshOn": "",
+                                            "redrawOn": "",
+                                            "dataGridLabel": false,
+                                            "labelPosition": "top",
+                                            "description": "",
+                                            "errorLabel": "",
+                                            "tooltip": "",
+                                            "hideLabel": false,
+                                            "tabindex": "",
+                                            "disabled": false,
+                                            "autofocus": false,
+                                            "dbIndex": false,
+                                            "customDefaultValue": "",
+                                            "calculateValue": "",
+                                            "calculateServer": false,
+                                            "widget": null,
+                                            "validateOn": "change",
+                                            "validate": {
+                                                "required": false,
+                                                "custom": "",
+                                                "customPrivate": false,
+                                                "strictDateValidation": false,
+                                                "multiple": false,
+                                                "unique": false
+                                            },
+                                            "allowCalculateOverride": false,
+                                            "encrypted": false,
+                                            "showCharCount": false,
+                                            "showWordCount": false,
+                                            "allowMultipleMasks": false,
+                                            "addons": [],
+                                            "id": "e8oejj"
+                                        },
+                                        {
+                                            "label": "Manage loan",
+                                            "action": "custom",
+                                            "showValidations": false,
+                                            "leftIcon": "fa fa-external-link",
+                                            "tableView": false,
+                                            "key": "manageLoan",
+                                            "type": "button",
+                                            "custom": "window.location.href = \"https://protege-secure.csnpe-nslsc.canada.ca/en/public/sign-in-method\";",
```

Comment:
I believe that we have been opening external links in a new tab. I missed this in previous PR for the other welcome page.